web-platform-when-why/index.html

```diff
@@ -0,0 +1,423 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>The Right Choice: When is the Web the right answer &amp;mdash;and why?</title>
+    <meta name="description" content="Going native or building for Web is a perennial hot topic in the mobile development space, but treating these alternatives as irreconcilable is naive. Nowadays, each option offers distinct opportunities and advantages. With the advent of new web technologies such as Service Workers, Push API and Web Manifest, let’s reconsider the Web as a competitive platform.">
```

Comment:
You might want to consider using an alternative to "go native", for the reasons I explained before. Also, the collective imperative in English doesn't mesh well with subordination, so I'd suggest "it's time to reconsider the Web as a competitive platform" or something like that.